env/test/test.properties

```diff
@@ -1 +1,6 @@
-OPERATOR_ENV=prod
\ No newline at end of file
+CHANNEL = ocp-4.4
+OPERATOR_VERSION = v1.0.1
+CSV = openshift-pipelines-operator.
+TKN_VERSION = 0.9.0
```

Comment:
Why do you need TKN_VERSION? Shouldn't tests use whatever is on `PATH`?

Reply:
Ah, I missed I am not using it probably I need to remove it